js/counter.js

```diff
@@ -1,16 +1,97 @@
 var PLAYER = {
-  points: 0
+  points: 0,
+  teenagers: 0,
+  kettles: 0,
+  theaters: 0
 }
 
 function earnPoints() {
   PLAYER.points += 1
   updateUI()
 }
 
+function buyTeenagers(num) {
+  teenagerIncrease = () => {
```

Comment:
Returning functions from a function! Nice. 🤓 

Only problem here is that the variable has been defined in the global scope because there's no `var` or `let` or `const` declaration keyword. (See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#Implicit_globals_and_outer_function_scope)